api/nodes/views.py

```diff
@@ -2382,6 +2382,8 @@ class NodeProvidersList(JSONAPIBaseView, generics.ListAPIView, NodeMixin):
     view_category = 'nodes'
     view_name = 'node-providers'
 
+    ordering = ('-pk',)
```

Comment:
Ah, thanks. Looks like `NodeProvider.pk == Node._id`  though, which is probably not what we want to sort on. What if we store `NodeProvider.id` which is the equal to `node.id` and sort on that instead?